cms/djangoapps/contentstore/models.py

```diff
@@ -0,0 +1,32 @@
+"""
+Models for contentstore
+"""
+
+from django.db.models.fields import TextField
+
+from config_models.models import ConfigurationModel
+
+
+class VideoEncodingDownloadConfig(ConfigurationModel):
```

Comment:
There's no current requirement to display this to the user.  Yes, it would only be through the django admin UI.